src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -347,6 +422,11 @@ module.exports = function<T, P, I, C>(config : HostConfig<T, P, I, C>, getSchedu
       return bailoutOnLowPriority(current, workInProgress);
     }
 
+    // If we don't bail out, we're going be recomputing our children so we need
+    // to drop our effect list.
+    workInProgress.firstEffect = null;
+    workInProgress.lastEffect = null;
```

Comment:
Correction. This is correct but I believe the problem is the TODO comment above in bailoutOnLowPriority. Needs to be fixed.